### PR TITLE
Added keymap/userspace for dz60 build 4 with readme

### PR DESCRIPTION
**Pull request by billiams**
## Description

Added custom keymaps for DZ60 build 4 along with readme and how-to.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
This PR supersedes #5665.

File: keyboards/dz60/keymaps/billiams/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+## Billiam's DZ60 layout
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/billiams/README.md]: Can you please rename this file so its name is fully lowercase?

e.g. `keyboards/dz60/keymaps/billiams/readme.md`

File: keyboards/dz60/keymaps/billiams/keymap.c [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#include QMK_KEYBOARD_H
+
+#define ______ KC_TRNS
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/billiams/keymap.c]: QMK defines an alias for this keycode automatically.

https://github.com/qmk/qmk_firmware/blob/d3268280ef7077ec1c800d30b8e98f1976192a14/quantum/quantum_keycodes.h#L29-L31

Please delete Line 3, and replace all instances of  
`______` (six underscores) with  
`_______` (seven underscores).

Comment (billiams) [reply keyboards/dz60/keymaps/billiams/keymap.c]: @noroadsleft okay done

File: keyboards/dz60/keymaps/billiams/README.md (hunk 1 above)
Comment (billiams) [reply keyboards/dz60/keymaps/billiams/README.md]: done

**Pull request merged**